pkg/processor/sourceprocessor/README.md

```diff
@@ -70,17 +70,25 @@ processors:
 
 ## Pod annotations
 
-The following [Kubernetes annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
-can be used on pods:
+The following [Kubernetes annotations][k8s_annotations_doc] can be used on pods:
 
-- `sumologic.com/exclude` - records from this pod are dropped
-- `sumologic.com/include` - overrides `exclude` processor settings, records from this pod are not checked against those regexes
+- `sumologic.com/exclude` - records from a pod that has this annotation set to
+  `"true"` (a string) or `true` (a boolean), are dropped,
+
+  **NOTE**: this has precedence over `sumologic.com/include` if both are set at
+  the same time for one pod.
+- `sumologic.com/include` - records from a pod that has this annotation set to
+  `"true"` (a string) or `true` (a boolean) are not checked against exclusion
```

Comment:
```suggestion
  `"true"` are not checked against exclusion
```
Kubernetes annotation values can only be string.

It's good you mention below that the OT attribute value can be either a string or a boolean :+1: 
But when talking about annotations, I think only talking about strings is justified.